app/services/export_service_base.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "csv"
+
+# Service to export data
+class ExportServiceBase
```

Comment:
Again, not seeing `ExportServiceBase` used more that one. Perhaps a premature abstraction?